app/views/health/careplans/_pctp_signatures.haml

```diff
@@ -1,31 +1,62 @@
 %article.health-care-plan__responsible-parties.l-print__section--page
   %h1 Responsible Parties Signatures
 
-  %dl.l-print__item
-    %dt.inline Patient:
-    %dd.inline= @patient.name
-    %dd.l-print__signature &nbsp;
-    %dt.inline Signed On:
-    %dd.inline= @careplan.patient_signed_on&.to_date
+  .signature__section
+    .signature__section__row
+      .signature__label Patient:
+      .signature__value= @patient.name
+    .signature__section__row
+      .signature__section__signature
+        .signature__label Patient Signature:
+        .signature__line
```

Comment:
If `patient_signed_on` is present, should we show something like `[DIGITALLY SIGNED]` or the patients name, instead of a blank line? Not sure what the workflow is.

Reply:
I like that idea.  How about `[DIGITALLY SIGNED on <date>]`

Reply:
oh, the date is already there.  `[DIGITALLY SIGNED]` then.